### ISSUE TITLE
Two-way data binding not working with <ion-input> in an electron app 

### ISSUE THREAD

**Issue by ssreekanth** [driftyco/ionic#7675]
#### Short description of the problem:

I'm using ionic-angular@2.0.0-beta.11, and i'm having trouble getting two-way binding working with `<ion-input>`. However, if I replace it with html `<input>` tag, then two-way binding works just fine. You can see it in the gif below -

![ion-input-issue-with-electron](https://cloud.githubusercontent.com/assets/1380334/17612239/93b3ebc6-6004-11e6-820b-5732f2469737.gif)
#### What behavior are you expecting?

Two-way data binding works with `<ion-input>` in an electron app.

**Steps to reproduce:**
1. clone [repo](https://github.com/ssreekanth/electron-ionic2-webpack-starter.git) `https://github.com/ssreekanth/electron-ionic2-webpack-starter.git`
2. `git checkout tags/ion-input-issue`
3. `npm install`
4. `npm start` 
5. wait for few seconds till webpack completes the build. 

**home.html**

``` html
<ion-header>
  <ion-navbar>
    <ion-title>
      Home
    </ion-title>
  </ion-navbar>
</ion-header>

<ion-content class="home">
  <ion-card>
    <ion-card-header>
      Using &lt;ion-input&gt; tag
    </ion-card-header>
    <ion-card-content>
      <ion-item>
        <ion-label primary floating>Name</ion-label>
        <ion-input type="url" [(ngModel)]="name"></ion-input>
      </ion-item>
    </ion-card-content>
  </ion-card>
  <ion-card>
    <ion-card-header>
      Using &lt;input&gt; tag
    </ion-card-header>
    <ion-card-content>
      <ion-item>
        <p>Name</p>
        <input [(ngModel)]="name"/>
      </ion-item>
    </ion-card-content>
  </ion-card>
  <ion-card>
    <ion-card-header>
      <ion-label>Hello {{name}}!</ion-label>
    </ion-card-header>
    <ion-card-content>
      <button (click)="showName()">Click Me</button>
    </ion-card-content>
  </ion-card>
</ion-content>
```

**home.ts**

``` js
import {Component} from "@angular/core";
import { AlertController } from "ionic-angular";

@Component({
  template: require("./home.html")
})

export class HomePage {
  name: string;
  constructor(private alertController: AlertController) {

  }
  showName() {
    console.log(this.name);
    let alert = this.alertController.create({
      title: "Hello " + this.name + "!",
      buttons: [" OK"]
    });
    alert.present();
  }
} 
```

**Which Ionic Version?** ionic-angular@2.0.0-beta.11

**Comment by ssreekanth** [comment]
Forum thread -  https://forum.ionicframework.com/t/two-way-data-binding-not-working-with-ion-input-in-an-electron-app/59955

**Comment by ssreekanth** [comment]
@jgw96 any update on this?

**Comment by jgw96** [comment]
Hello! Sorry for the delay on this. Unfortunately I cannot seem to reproduce this issue. Could you start a brand new fresh electron project and test again? The issue could be that you are using an older version of electron that may be bugged. Also, would you mind trying a plain Angular 2 app in electron and seeing if `ngModel` works there? Thanks for using Ionic!

**Comment by jgw96** [comment]
Hello all! As it seems it has been a while since there was any activity on this issue i will be closing it for now. Feel free to comment if you are still running into this issue. Thanks for using Ionic!

**Comment by ssreekanth** [comment]
@jgw96 I'm still running into this issue with the latest version of electron. As I mentioned in the issue description, `ngModel` works fine with plain `<input>`. The issue is with `<ion-input>`. Wondering how it worked for you. Could you share your sample code, that worked for you? If not, could you see what is wrong in my code (you can find link to my repo in the `steps to reproduce` section in the bug description).

**Comment by ssreekanth** [comment]
@jgw96 could you please reopen the issue, as i'm still running into this?

**Comment by marwesfur** [comment]
I can confirm that. We're building a package of (higher level) components with Ionic for use in a web and mobile app and this issue is really a show stopper. Thanks for making Ionic even greater :)

**Comment by ssreekanth** [comment]
@jgw96 Did ionic team get a chance to look into this issue?

**Comment by jgw96** [comment]
Hello @ssreekanth would you mind trying this with the rc.0 release? [Here](https://github.com/driftyco/ionic/blob/master/CHANGELOG.md#200-rc0-2016-09-28) is a link to the changelog that includes instructions for how to update your app. Thanks!

**Comment by ssreekanth** [comment]
@jgw96 Sure, I'll try updating my app to use ionic rc.0 version and let you know how it goes.

**Comment by ssreekanth** [comment]
I have trouble building app using ionic 2.0.0-rc.0 with webpack. Blocked on https://github.com/driftyco/ionic/issues/8251

**Comment by jgw96** [comment]
Hello @ssreekanth ! Now that the app-scripts issue has been fixed is this still an issue?

**Comment by ssreekanth** [comment]
I'll give it a shot again and update you how it goes..

**Comment by ssreekanth** [comment]
I still can't get Electron app build with ionic 2 rc2. Reported a new issue #9050 for help regarding boilerplate for Electron app with Ionic 2.

Blocked on #9050.

**Comment by jgw96** [comment]
Going to close this one for now as a duplicate of #9050
